.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,7 +6,7 @@ about: Create a report to help reproduce and fix the issue
 ## Round ID:
 
 <!--- **INCLUDE THE ROUND ID**
-If you discovered this issue from playing tgstation hosted servers:
```

Comment:
Well, technically we're still a **tgstation hosted** server so we should leave this here,